src/cdk/utils/basic-authorizer-handler/index.ts

```diff
@@ -39,10 +39,27 @@ function createAllowPolicy(
   };
 }
 
+export function getAuthHeaderValue(headers?: {
+  [name: string]: string;
```

Comment:
I would prefer the signature to be `getAuthHeaderValue(headers: Record<string, string> | undefined): string | undefined` instead of the optional parameter. Calling this function without passing an argument makes no sense for me.

Reply:
Fixed